avm/res/network/public-ip-address/main.bicep

```
@@ -52,16 +52,19 @@ param ddosSettings ddosSettingsType?
 param location string = resourceGroup().location
 
 @description('Optional. Array of role assignment objects that contain the \'roleDefinitionIdOrName\' and \'principalId\' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: \'/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11\'.')
-param roleAssignments roleAssignmentType?
+param roleAssignments roleAssignmentType
 
 @description('Optional. Enable/Disable usage telemetry for module.')
 param enableTelemetry bool = true
 
+@description('Optional. The idle timeout of the public IP address. Default is `4`.')
```

```suggestion
@description('Optional. The idle timeout of the public IP address.')
```
Redundant and may be outdated eventually